src/app/contextBroker/contextBroker.cpp

```diff
@@ -376,6 +377,9 @@ PaArgument paArgs[] =
 #define ETT_COMPS_V2            2, { "v2", "types" }
 #define ETT_COMPS_WORD          ""
 
+#define GAS                     GetAllSubscriptionsRequest
```

Comment:
*GET* should not be a part of the 'request name'.
What if we add a POST or PUT ... ?

I would propose **SubscriptionRequest**, unless it's already taken, of course ...